### PR TITLE
Fix/hpc us

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

## Description

Batch for HPC in the US

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable.controller.js [hunk 1]
```diff
@@ -96,6 +96,9 @@ angular.module('App').controller(
         )
         .then((contracts) => {
           this.bindings.contracts.value = contracts;
+          if (!contracts.length) {
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable.controller.js:99]:
```suggestion
          if (isArray(contracts) && !isEmpty(contracts)) {
```

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable.html [hunk 2]
```diff
@@ -43,6 +43,7 @@
                 </oui-datagrid>
 
                 <div
+                    data-ng-if="$ctrl.bindings.contracts.value.length > 0"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable.html:46]:
```suggestion
                    data-ng-if="!$ctrl.allContractsSigned"
```
and define and initialize allContractsSigned in the controller

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html [hunk 3]
```diff
@@ -12,7 +12,28 @@
...
-        <p data-translate="dedicatedCloud_close_service_desc"></p>
-        <p data-translate="dedicatedCloud_close_service_question"></p>
+        <form
+            data-ng-if="$ctrl.canDeleteAtExpiration"
+            class="mt-4"
+            name="cancelSubscriptionForm"
+            novalidate
+        >
+            <oui-field
+                label="{{:: 'server_close_service_form_label' | translate }}"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html:22]:
```suggestion
                data-label="{{:: 'server_close_service_form_label' | translate }}"
```

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html [hunk 4]
```diff
@@ -12,7 +12,28 @@
...
+            >
+                <oui-select
+                    required
+                    items="$ctrl.TERMINATE_OPTIONS"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html:26]:
```suggestion
                    data-items="$ctrl.TERMINATE_OPTIONS"
```

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html [hunk 5]
```diff
@@ -12,7 +12,28 @@
...
+                    model="$ctrl.model.termination"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html:27]:
```suggestion
                    data-model="$ctrl.model.termination"
```

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html [hunk 6]
```diff
@@ -12,7 +12,28 @@
...
+                    match="label"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html:28]:
```suggestion
                    data-match="label"
```

**Comment by marie-j** [comment]
@radireddy Thanks for your review but this is an aggregation of PR that were already reviewed

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html (hunk 3 above)
Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html:22]: As mentioned by @antleblanc https://github.com/ovh/manager/pull/3466#discussion_r472802183 We are shifting on this behavior. Only for webcomponents, meaning that on <oui-field>, we are ok to stop using them.

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable.controller.js (hunk 1 above)
Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable.controller.js:99]: This was already approved https://github.com/ovh/manager/pull/3476

**Review by Jisay — APPROVED** [review]

**Review by lizardK — APPROVED** [review]

**Pull request merged**